packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -829,13 +829,13 @@ export class LocalFluidDataStoreContextBase extends FluidDataStoreContext {
         });
     }
 
-    public generateAttachMessage(): IAttachMessage {
+    public async generateAttachMessage(): Promise<IAttachMessage> {
```

Comment:
This API existed for two reasons:
1. synchronous nature of workflows that depended on it
2. some subtle differences in content returned from async summarize methods.
Given that # 1 is gone in this PR, it would be great to have a follow up issue to examine # 2 and see if we can substantially reduce code duplication. I.e. make this logic rely as much as possible on summarization flow, only adding (if needed) subtle differences that are required (if they are) for attachment workflows